examples/reactjs/hooks/sample-app/public/index.html

```diff
@@ -26,26 +26,23 @@
     -->
     <title>React App</title>
     <script>
-      window.rudderanalytics = [];
-      var methods = [
-        'load',
-        'page',
-        'track',
-        'identify',
-        'alias',
-        'group',
-        'ready',
-        'reset',
-        'getAnonymousId',
-        'setAnonymousId',
-        'getUserId',
-        'getUserTraits',
-        'getGroupId',
-        'getGroupTraits',
-        'startSession',
-        'endSession',
-        'getSessionId',
-      ];
+      window.RudderSnippetVersion = '3.0.1';
+    window.rudderanalytics = [];
```

Comment:
The formatting is a bit off.

Reply:
prettier is not working for html files, I will check

Reply:
addressed